lang/sv.json

```diff
@@ -61,6 +61,7 @@
    "CoC7.UnderlyingInsanity":"Underliggande galenskap",
    "CoC7.TemporaryInsanity":"Temporär galenskap",
    "CoC7.IndefiniteInsanity":"Varaktig galenskap",
+   "CoC7.NotInsane": "Ingen",					 
    "CoC7.UnlockActor":"Lås up karaktär",
```

Comment:
Spelling error: "Lås up karaktär" should be "Lås upp karaktär"

Reply:
Thanks! Tack så mycket!